code/script.js

```diff
@@ -29,23 +24,24 @@ const showMessage = (message, sender) => {
       </section>
     `
   }
-  // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
   chat.scrollTop = chat.scrollHeight
 }
 
-// Starts here
 const greetUser = () => {
-  // here we call the function showMessage, that we declared earlier with the argument "Hello there, What's your name?" for message, and the argument "bot" for sender
-  showMessage("Hello there, What's your name?", 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+  showMessage("Hello there, What's your name?", 'bot')  
 }
+const handleNameInput = event => {
+  event.preventDefault();
+  userName = nameInput.value;
+  nameInput.value = "";
+  showMessage( `${userName}`, `user`);
+  setTimeout(reply, 1000)
+}
+
+nameForm.addEventListener (`submit`, handleNameInput);
 
-// Set up your eventlisteners here
+function reply () {
```

Comment:
This code appears when I answer with my name, but, there it's stops, it's not cristal clear what you can ask for here, but since it is a work in progress, you can continue with it later on